packages/@aws-cdk/aws-apigateway/lib/json-schema.ts

```diff
@@ -63,7 +63,7 @@ export interface JsonSchema {
   readonly minProperties?: number;
   readonly required?: string[];
   readonly properties?: { [name: string]: JsonSchema };
-  readonly additionalProperties?: JsonSchema;
+  readonly additionalProperties?: JsonSchema | boolean;
```

Comment:
Sadly we can’t allow unions in public APIs because they don’t translate well to some languages. I am wondering why isn’t this just a Boolean? What’s the use case for specifying a jsonSchema here?

Reply:
It can be and also what I am use for. I didn't want to break anything without knowing how other people use it. I will change it now.